react/src/views/BoxEdit/components/BoxEdit.tsx

```diff
@@ -127,6 +127,10 @@ function BoxEdit({
     tags: boxData?.tags || [],
   };
 
+  if (boxData?.product?.deleted !== null) {
```

Comment:
I wasn't sure where to add typing information about this new property. I see these types get pulled directly from the generated graphql.ts file, but that file auto-generates it as `any`.

Is there a way to be more specific about the types? @vahidbazzaz

Reply:
@HaGuesto